contracts/core/EntryPointSimulations.sol

```diff
@@ -192,7 +192,7 @@ contract EntryPointSimulations is EntryPoint, IEntryPointSimulations {
 
     //slightly stricter gas limit than the real EntryPoint
     function _getVerificationGasLimit(uint256 verificationGasLimit) internal pure virtual override returns (uint256) {
-        return verificationGasLimit - 300;
+        return verificationGasLimit - 350;
```

Comment:
Magic number. As you need to change it, please extract a constant.

Reply:
unrelated. we're going to remove "Simulation" from the critical path, and thus all this code will be removed.